pkg/kubefed2/unjoin.go

```diff
@@ -345,7 +345,7 @@ func deleteClusterRoleAndBinding(clusterClientset client.Interface, saName, name
 	}
 
 	roleName := util.RoleName(saName)
-	commonClusterRoleName := util.CommonClusterRoleName(saName)
+	commonClusterRoleName := util.HealthCheckRoleName(saName)
```

Comment:
sorry, one more nit: `s/commonClusterRoleName/healthCheckRoleName/`

Reply:
My bad, I should have caught this. I'll change.